x/incentives/keeper/distribute.go

```diff
@@ -379,39 +379,33 @@ func (k Keeper) distributeSyntheticInternal(
 	return k.distributeInternal(ctx, gauge, sortedAndTrimmedQualifiedLocks, distrInfo)
 }
 
-// syncGroupGaugeWeights updates the individual and total weights of the gauge records based on the splitting policy.
-// It mutates the passed in object and sets the updated value in state.
-// If there is an error, the passed in object is not mutated.
-//
-// It returns an error if:
-// - the splitting policy is not supported
-// - a lower level issue arises when syncing weights (e.g. the volume for a linked pool cannot be found under volume-splitting policy)
-func (k Keeper) syncGroupGaugeWeights(ctx sdk.Context, groupGauge types.GroupGauge) error {
-	if groupGauge.SplittingPolicy == types.Volume {
-		err := k.syncVolumeSplitGauge(ctx, groupGauge)
+// syncGroupWeights updates the individual and total weights of the group records based on the splitting policy.
+func (k Keeper) syncGroupWeights(ctx sdk.Context, group types.Group) error {
+	if group.SplittingPolicy == types.Volume {
+		err := k.syncVolumeSplitGauge(ctx, group)
 		if err != nil {
 			return err
 		}
 	} else {
-		return types.UnsupportedSplittingPolicyError{GroupGaugeId: groupGauge.GroupGaugeId, SplittingPolicy: groupGauge.SplittingPolicy}
+		return types.UnsupportedSplittingPolicyError{GroupGaugeId: group.GroupGaugeId, SplittingPolicy: group.SplittingPolicy}
 	}
 
 	return nil
 }
 
-// syncVolumeSplitGauge syncs a group gauge according to volume splitting policy.
+// syncVolumeSplitGauge syncs a group according to volume splitting policy.
 // It mutates the passed in object and sets the updated value in state.
 // If there is an error, the passed in object is not mutated.
 //
 // It returns an error if:
 // - the volume for any linked pool is zero or cannot be found
 // - the cumulative volume for any linked pool has decreased (should never happen)
-func (k Keeper) syncVolumeSplitGauge(ctx sdk.Context, groupGauge types.GroupGauge) error {
+func (k Keeper) syncVolumeSplitGauge(ctx sdk.Context, groupGauge types.Group) error {
```

Comment:
Thoughts on renaming this to `syncVolumeSplitGroup`?

Reply:
Agreed

Reply:
Missed this change but tracking here: https://github.com/osmosis-labs/osmosis/issues/6501